example_scenes/advanced_tex_fonts.py

```diff
@@ -0,0 +1,691 @@
+import os
+from manim.utils.tex import TexTemplateFromFile
+from manim.mobject.svg.tex_mobject import TextMobject
+from manim.utils.config_ops import digest_config
+
+# for the demo Scene
+from manim import (
+    BLACK,
+    WHITE,
+    UP,
+    UL,
+    ORIGIN,
+    DOWN,
+    LEFT,
+    RIGHT,
+    TextMobject,
+    ApplyMethod,
+    AnimationGroup,
+    VGroup,
+    FadeIn,
+    FadeOut,
+    Transform,
+    FadeOutAndShift,
+    FadeInFromDown,
+    FadeOutAndShiftDown,
+    Scene,
+)
```

Comment:
I suggest you to use
```suggestion
from manim import *
```
as in other examples.